src/attributes.ts

```diff
@@ -644,6 +644,19 @@ const ATTRIBUTES_DEFINITION_BASE = {
 		name: 'A chainable armbinder',
 		description: 'An armbinder that is chainable to another item',
 	},
+	// Jute Rope
+	Waist_rope_anchor_point: {
+		name: 'Rope around waist',
+		description: 'A rope around the waist',
+	},
+	Vulva_rope_anchor_point: {
+		name: 'Crotch rope link',
+		description: 'A rope linking crotch rope to tied hands',
+	},
+	Thight_rope_anchor_point: {
+		name: 'Crotch rope link',
+		description: 'A rope linking crotch rope to tied thighs',
+	},
```

Comment:
Suggested wording that I feel better matches the current usage:
Name:  Thighs rope
Description: A rope over thighs that a linking rope can be tied to

Context in which it is used currently (when trying to select the option requiring this, not yet having it selected):
> The crotch jute ropes requires "A rope linking crotch rope to tied thighs" (must be worn under the crotch jute ropes).

After suggestion:
> The crotch jute ropes requires "A rope over thighs that a linking rope can be tied to" (must be worn under the crotch jute ropes).